package.json

```diff
@@ -536,7 +536,7 @@
       },
       {
         "command": "todo.toggleBox",
-        "title": "Todo: Toggle Box"
+        "title": "Todo: Toggle CheckBox"
```

Comment:
I can't merge this line, maybe "checkbox" is more appropriate than "box", but it's probably to late to fix this. Besides you're only updating one instance, all instances of "box" should be updated to "checkbox" if one wanted to made the change.

Reply:
I did the name change from box to checkbox only in the title because it is what is seen in the command pallete.

Reply:
Yeah but we should always use "CheckBox" then, meaning `toggleBox` should be `toggleCheckBox`, the readme should be updated, and many many more little changes.

Reply:
Ok. Do what you think is the best.

Reply:
Can you remove this line from the PR?